### PR TITLE
Adds some fennec treats

### PR DESCRIPTION
**Pull request by Ketrai**
Adds an assortment of fennec treats that are primarily ordered through cargo.

Content list :

Jerboa
- Scorpions / cooked
- Grubs / Candied, pink, blue, and purple
- Locusts / fried
- Honey ant / sliceable
- Honey candy

I might do packs for other species at some point, but crate code is weird.

I would also give a random chance for one of these to very rarely spawn in maint, but it seems there's a few mapping changes in the works, and fixing conflicts just for one little tiny addition feels a bit meh.
image
image

### CODE REVIEW

**Comment by dwinters99** [comment]
You wouldn't need to change the map to add them, simply edit the maint loot table.

Additionally, some of the sprites seem quite garish and don't match the style of quality of other food items. Also, you should probably include pictures of the sprites in the main post.

**Comment by Ketrai** [comment]
> You wouldn't need to change the map to add them, simply edit the maint loot table.
> 
> Additionally, some of the sprites seem quite garish and don't match the style of quality or other food items. Also, you should probably include pictures of the sprites in the main post.

image
I think you should really watch your tone before you start calling sprites "garish and undercooked". I've looked at your PR history, you're not much of a spriter. The majority of icon prs are just re-using existing sprites of the codebase, rather than making new unique ones. Just because I like cartoonish sprites for food, doesn't automatically make them improper. Loads of food sprites are literally just scaled down PNGs because whoever implemented them didn't feel like making a proper sprite, so I wouldn't praise quality too highly.

- We don't have a style guide, it doesn't matter what style sprites use.
- People of any skill level are encouraged to make contributions to the code, how are they supposed to get better at spriting if they don't make any out of fear of people like you.
- My sprites vary in quality, I do little vorestation contributions for fun. I don't get paid to do this, so they're not all going to be the best of the best I can manage. (E.g monkfish fillet vs sharkmeat chunk. One's really carefully shaded because I felt like practicing, the other is messily shaded because I didn't feel like putting in the effort.)
- Please know what you're talking about before you start throwing flat critique at people without nuance.

Just so you know, I made this. I don't like to pull the "look at what I made" card often, but folks like you just grind my gears.
https://pbs.twimg.com/media/ElMSy8XW0AASA2I?format=jpg&name=4096x4096

**Comment by dwinters99** [comment]
I seem to have struck a nerve. I was being reserved and trying to be constructive, so allow me to give you more of my personal thoughts.

> We don't have a style guide, it doesn't matter what style sprites use.

You're right! In fact, part of the reason I love SS13's "style" is how different and disparate the different icons are.

> People of any skill level are encouraged to make contributions to the code, how are they supposed to get better at spriting if they don't make any out of fear of people like you.

My mistake, I should've taken more time to be more constructive! Just like the example you set when I showed off my first ever spritework!
image
Additionally, this point seems to be incapable of co-existing with this statement. 
>I've looked at your PR history, you're not much of a spriter.

> Please know what you're talking about before you start throwing flat critique at people without nuance.

To quote someone whose name I don't remember, "I don't need to be a pilot to know helicopters don't belong in trees."

In conclusion, I *WAS* trying to be vague as not to trigger this exact defensiveness. Considering how little I actually said, it seems you aren't willing to take criticism. I understand that you consider yourself a something of a big-time spriter, but not everyone does everything perfectly every time. Additionally, the "look what I made" doesn't exactly make much sense (to me) considering I never accused YOU of making poor sprites, just that these specifically were not my favorite. You seem to be extremely defensive, so I'd recommend reflecting on that and taking things like this more in stride.

**Comment by Ketrai** [comment]
Yeah. I used to be more even more crass. Nowadays I try to point people at examples they can use to improve. You also lifted that comment out of context. If someone ties a gas lamp to their back, I'm allowed to laugh. Not because I think the sprite quality is bad, it corresponds to the original gas lamp. But because it's a silly thing. Apologies if that came off as a stab at you as a spriter back then, it wasn't ever meant that way. But misunderstandings do happen. Also I don't remember seeing your name on discord since you use a vastly different name on github. It makes you come off as a random person. Now as for your notes.

I'm not certain which sprites in specific you're talking here other than the scorpion or ant. It's pretty hard to draw an ant facing right towards you, and whilst the scorpion isn't really facing any specific direction, I like to go with a pose that shows off the object. I did consider adding some code to make it more interactible or turning it into a simple mob, but I don't like delving that deep into the coding side or things. I'm not that experienced when it comes to writing out lines. Just so you know. I did animate the scorpion. A very simple little wiggle to make it feel alive. Tbf I don't think I've ever seen this be much of a concern before. If it doesn't move around in the world or animate players tend to consider something to be dead. Unless they want to involve it in a play. Then they'll happily treat it as alive. 

Otherwise I really don't have any idea what you're talking about or trying to point me at. You went from "garish, not the same quality as other foods." To "weird perspective, looks like a painting and not interactible enough despite looking alive?" 

Trust me, I don't mind critisism. I can point out a dozen things that are wrong and I could improve on with the picture above. But I want to be able to at least follow along with what the other person is saying.

**Comment by Ketrai** [comment]
I'm also not saying over-nuacing your critisism is fine either. (Not talking about the critisism given above as an example). Since that in turn can look intimidating. I feel the safest approach is to pull a compliment sandwich or to grab some examples and a red marker to point out... your point. A compliment sandwich is simply starting off by mentioning something you like, then something you don't, and finally something you like again. Vizualising and a soft approach tends to yield better reactions.
